pubpub/pubpub
# update/fix: Use publisher for copyright

Ended up being a larger one than I anticipated. This PR:

- Uses the publisher field for the Pub copyright license text, if filled out
- Fixes the copyright license display for PDF and JATS
- Fixes(?) a bug where the copyright date for a book was always displaying the prior year due to timezone issues
- Resolves #596 by adding images to cc-by licenses in the pub footer, because why not while I was down there?

Flew a little blind here and had to add some data from up in the rendering tree, so definitely open to suggestions of better ways to organize. But I think it's pretty straightforward.

<img width="702" alt="Screen Shot 2021-12-01 at 16 56 48" src="https://user-images.githubusercontent.com/639110/144321500-b2a32e3d-3137-48c7-8e76-42d31865ec9b.png">
<img width="703" alt="Screen Shot 2021-12-01 at 16 56 42" src="https://user-images.githubusercontent.com/639110/144321508-6a0df1e8-026b-4801-b949-0bf1443a140b.png">
<img width="482" alt="Screen Shot 2021-12-01 at 17 05 01" src="https://user-images.githubusercontent.com/639110/144321548-8439d2ae-b65c-4969-bcb1-0ad3d7d6db3a.png">
<img width="527" alt="Screen Shot 2021-12-01 at 17 06 32" src="https://user-images.githubusercontent.com/639110/144321766-26b7d659-27ad-40b6-abb5-1c75143e1787.png">

_Test Plan_
1. Create a pub in a community that has the premium license flag turned on and no publisher set in settings, and set the license of that pub to copyright.
2. Make sure the license is `Copyright (c) {Community Title} {Pub Creation Date}. All rights reserved.`
3. Download PDF and JATS and make sure the license matches.
4. Add a publisher to community settings. Visit the pub and make sure the license is: `Copyright (c) {Publisher} {Pub Creation Date}. All rights reserved.`
5. Check the PDF and JATS to make sure they match (you will have to make body changes to regen).
6. Add the pub to a public book collection with a copyright year different from this year set (e.g. 2020). Make sure the license is: `Copyright (c) {Community Title} {Book Copyright Year}. All rights reserved.`
7. Check the PDF and JATS to make sure they match (you will have to make body changes to regen).
8. Add the pub to a public issue collection with a publication year different from this year set (e.g. 2020). Make sure the license is: `Copyright (c) {Community Title} {Pub Release Year}. All rights reserved.`
9. Check the PDF and JATS to make sure they match (you will have to make body changes to regen).
10. Remove the pub from those collections and release it, setting a custom release date with a year different than this one (e.g. 2020). Make sure the license is: `Copyright (c) {Community Title} {Custom Publication Date Year}. All rights reserved.`
11. Check the PDF and JATS to make sure they match (you will have to make body changes to regen).
